packages/autorest.typescript/test/rlcIntegration/generated/httpInfrastructureRest/package.json

```diff
@@ -35,7 +35,7 @@
       },
       {
         "path": "src/httpInfrastructureRestClient.ts",
-        "prefix": "package-version"
```

Comment:
i remember we have also package-version in readme file, could you double confirm the correctness of that config?

Reply:
offline confirmed this is correct.